functions.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**A function that connects to the SQL database and retrieves an object.
+ * @return object
+ */
+function getDb(): object {
+    return new PDO ('mysql:host=db; dbname=sami-collection','root', 'password');
+}
+
+/**This function takes a database object and selects the items from the database to output an array.
+ * @param object $db is the object from the getDb function that is a connection to the database content.
+ * @return array this returns an array based on the selection criteria within the prepare statement.
+ */
+function getItems(object $db): array {
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $query =  $db->prepare("SELECT `name`, `designer`, `style`, `year_released`, `image` FROM `items` ORDER BY `id`;");
+    $query->execute();
+    return $query->fetchAll();
+}
+
+/**This function receives an array from the database selection and outputs a string that will receive styling.
+ * @param array $items is an array list of the selected criteria from the database query.
+ * @return string for each array entry a string is output that will be displayed in HTML.
+ */
+function arraySplit(array $items) : string{
```

Comment:
a more semantic name would be generateJacketMarkup

Reply:
also $items should be $jacket

Reply:
Have done this